src/components/SEO/SEO.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { useStaticQuery, graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+
+const SEO = ({ postData, frontmatter = {}, postImage, isBlogPost }) => {
+  const { site } = useStaticQuery(
+    graphql`
+      {
+        site {
+          siteMetadata {
+            title
+            description
+            siteUrl
+            image
+            author {
+              name
+            }
+            organization {
+              name
+              url
+              logo
+            }
+            social {
+              twitter
+              fbAppID
+            }
+          }
+        }
+      }
+    `,
+  );
+
+  const { siteMetadata: seo } = site;
+
+  const postMeta =
+    frontmatter || postData.childMarkdownRemark.frontmatter || {};
+
+  const title = postMeta?.title || seo?.title;
+  const description = postMeta?.description || seo?.description;
+  const image = postImage ? `${seo.siteUrl}${postImage}` : seo?.image;
+  const url = postMeta.slug ? `${seo.siteUrl}/${postMeta.slug}/` : seo?.siteUrl;
+
+  // const datePublished = isBlogPost ? postMeta.datePublished : false;
+
+  return (
+    <React.Fragment>
```

Comment:
We can leverage the shorthand `<>` and matching closing shorthand `</>` for the fragment here. I believe this is in use on the `Layout.js` component as well!